numpy/fft/_pocketfft.py

```diff
@@ -113,9 +113,9 @@ def fft(a, n=None, axis=-1, norm=None):
     axis : int, optional
         Axis over which to compute the FFT.  If not given, the last axis is
         used.
-    norm : {None, "ortho"}, optional
+    norm : {None, "ortho", "inverse"}, optional
         .. versionadded:: 1.10.0
-
+        .. versionadded:: 1.20.0
```

Comment:
The first versionadded is about the `norm` kwarg itself. The second is about `"forward"` (inverse). So I think we should put this at the very end of the paragraph as:
```
.. versionadded:: 1.20
    The "forward" option was added
```
(or similar, not quite sure how that renders).  I guess the order does not matter too much, depending on how it renders.

Reply:
The CI is failing here, because there needs to be a blank line after the tag.

Reply:
done

Reply:
the CI is still failing, but I don't know if it's still because of some issue here; I did add blank lines and the new `versionadded` as suggested, let me know if more changes are needed

Reply:
Ah, damn, no the test failure is unrelated. I merged another PR earlier today, which clashed with a deprecation introduced after that PRs tests ran. I will fix that later, sorry about that.  For documentation mainly CircleCI matters (It does not show up right now, not sure whats going on there).